README.md

```diff
@@ -1 +1,93 @@
-# SFPortfolioApp
\ No newline at end of file
+# SFPortfolioApp
+
+[![Swift Version][swift-image]][swift-url] [![License][license-image]][license-url] ![Platform]
+
+**SFPortfolioApp** is a catalogue containing various and reusable UI components. This is a centralised place for storing: <br>
+- [x] Common UI elements
+- [x] Tricky disigner wishes
+- [x] Time consuming views
+- [x] Interesting and unique components
+
+Table of contents
+=================  
+* [Requirements](#requirements)
+* [Components](#components)
+  * Asymmetric Collection View
+  * Synchronised Collection View
+  * Tooltip View
+  * Execution Button
+  * Segmented Control
+  * Shimmer Loading View
+  * Bottom Sheet
+* [Contacts](#contacts)
+* [More Projects](#more-projects)
+* [License](#license)
+
+## Requirements
+- iOS 14.1+
```

Comment:
Should we say that the iOS version varies for the different components instead of saying that is requires 14.1+ for all components. What do you think?

Reply:
This suggestion is good. It would be even better if we could provide iOS 13+ version of the same component. At least add tag in every element saying what is the minimum supported version in the list view.

Reply:
Reworked with required version for components is marked now in the table of contents. 14.1+ is set for the application because of usage of some SwiftUI functions like navigationTitle(_:) and so on.